src/librustc_codegen_llvm/debuginfo/namespace.rs

```diff
@@ -11,6 +10,7 @@ use crate::common::CodegenCx;
 use rustc::hir::def_id::DefId;
 use rustc::hir::map::DefPathData;
 
+use rustc_monomorphize::Instance;
```

Comment:
Isn't `Instance` a reexport of something from `rustc`? We should probably refer to the original in `rustc` noawadays.

Reply:
Ah, yes. It is a re-export of `rustc::ty::Instance`. Should I just remove the re-export altogether?

Reply:
I went ahead and removed it. Let me know what you think.